testutil/compose/README.md

```diff
@@ -31,11 +31,13 @@ Install the `compose` binary:
 # From inside the charon repo
 go install github.com/obolnetwork/charon/testutil/compose/compose
 
+# If on Mac M1, add the following to your bashfile (`.zshrc`, `.bashrc`):
+# export PATH="$PATH:$(go env GOPATH)/bin/darwin_amd64"
```

Comment:
why amd64?

Reply:
correct what @dB2510 says M1 is darwin_m1 or darwin_arm64 or something like that

Reply:
darwin_arm64: https://github.com/prysmaticlabs/prysm/pull/10192/files